lib/kafo/param.rb

```diff
@@ -23,6 +23,7 @@ def value
 
     def value=(value)
       @value_set = true
+      value      = value.to_s if value.class == ::HighLine::String  # don't persist highline extensions
```

Comment:
`value.is_a?(::HighLine::String)` looks a bit more readable to me and is a bit safer since it covers all children classes (would be same as <= :HighLine::String)